addon/components/sl-alert.js

```diff
@@ -65,18 +76,18 @@ export default Ember.Component.extend( TooltipEnabled, {
     /**
      * Whether to make the alert dismissable or not
      *
-     * @property {boolean} dismissable
-     * @default  false
+     * @type {Boolean}
+     * @default
```

Comment:
@default can be removed for non-complex data types due to the intended use of https://github.com/notmessenger/jsdoc-plugins#defaulttag